recipes/install_base_apache.rb

```diff
@@ -9,60 +11,58 @@
 when 'freebsd'
   packages = %w[apr pcre-8.33 libxml2 curl]
 else
-  Chef::Log.fatal("Unsupported platform: #{node[:platform_family]}.")
-  fail 'mod_security cookbook does not support this platform'
+  Chef::Log.fatal("Unsupported platform: #{node['platform_family']}.")
+  raise 'mod_security cookbook does not support this platform'
 end
 packages.each { |p| package p }
 
 # FIXME: ignoring lua for right now
 # make optional in the future
 
-directory node[:mod_security][:dir] do
+directory node['mod_security']['dir'] do
   recursive true
 end
 
-
-if node[:mod_security][:from_source]
+if node['mod_security']['from_source']
   # COMPILE FROM SOURCE
 
   # install required libs
 
-  case node[:platform_family]
+  case node['platform_family']
   when 'arch'
     # OH NOES
   when 'rhel', 'fedora', 'suse'
     package 'httpd-devel'
-    if node[:platform_version].to_f < 6.0
+    if node['platform_version'].to_f < 6.0
```

Comment:
It would be be better to compare ints, so should this be updated to `if node['platform_version'].to_i < 6` please